web-api/src/business/useCases/editRemoteStatusInteractor.ts

```diff
@@ -0,0 +1,49 @@
+import { Case } from '@shared/business/entities/cases/Case';
+import {
+  ROLE_PERMISSIONS,
+  isAuthorized,
+} from '@shared/authorization/authorizationClientService';
+import { ServerApplicationContext } from '@web-api/applicationContext';
+import { UnauthorizedError } from '@web-api/errors/errors';
+import { UnknownAuthUser } from '@shared/business/entities/authUser/AuthUser';
+import { getCaseByDocketNumber } from '@web-api/persistence/postgres/cases/getCaseByDocketNumber';
+import { updateCaseAndAssociations } from '@web-api/business/useCaseHelper/caseAssociation/updateCaseAndAssociations';
+import { withLocking } from '@web-api/persistence/postgres/utils/mutex';
+
+/**
+ * used for setting a date to indicate when a motr was granted
+ * @param {object} applicationContext the application context
+ * @param {object} providers the providers object
+ * @param {string} providers.timestamp timestamp when the motr was granted
+ * @param {string} providers.docketNumber the docket number
+ * @returns {object} the case data
+ */
+export const editRemoteStatus = async (
+  _applicationContext: ServerApplicationContext,
+  { docketNumber }: { docketNumber: string },
+  authorizedUser: UnknownAuthUser,
+) => {
+  if (!isAuthorized(authorizedUser, ROLE_PERMISSIONS.EDIT_DOCKET_ENTRY)) {
+    throw new UnauthorizedError('Unauthorized');
+  }
+
+  const caseToUpdate = await getCaseByDocketNumber({
+    docketNumber,
+  });
+
+  const caseEntity = new Case(caseToUpdate, { authorizedUser });
+
+  const updatedCase = await updateCaseAndAssociations({
+    authorizedUser,
+    caseToUpdate: caseEntity,
+  });
+
+  return new Case(updatedCase, { authorizedUser }).validate().toRawObject();
+};
+
+export const editRemoteStatusInteractor = withLocking(
```

Comment:
We don't seem to be using this interactor, instead updating the case on the front end and then using `updateCaseDetailsInteractor`. (If we did use this interactor, it wouldn't work since it does not currently update the `remoteTrialGranted` or `remoteTrialGrantedDate` fields on the case)